core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1993,6 +1993,9 @@ secret_configs:
     kubeconfig:
       field: sa.ci-operator.build01.config
       item: build_farm
+    sa.ci-operator.build01.token.txt:
```

Comment:
Are these changed? They appear in my files already. If these aren't changed can we squash the secrets ?